checkout-ui-custom/src/partials/Deliver/utils.js

```diff
@@ -557,6 +563,9 @@ export const submitDeliveryForm = async (event) => {
     for (let i = 0; i < fields.length; i++) {
       if (!address[fields[i]]) fullAddress[fields[i]] = $(`#bash--input-tv_${fields[i]}`).val();
       tvData[fields[i]] = $(`#bash--input-tv_${fields[i]}`).val();
```

Comment:
`tvData[fields[i]] = $(`#bash--input-tv_${fields[i]}`).val() || "";`

Easier and applies inline